tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -68,6 +71,24 @@ Operator Should Be Uninstalled
   [Arguments]  ${operator}
   Page Should Not Contain Element  //a[@data-test-operator-row="${operator}"]
 
+
+Switch To New Tab
+    [Arguments]  ${tabname}
+     Click Element        //a[contains(text(), "${tabname}")]
```

Comment:
can this be used as common keyword?

Reply:
Totally we could do that. May be in refactoring process we could do that.